test/test-kitchen.ts

```diff
@@ -66,140 +62,138 @@ const execOpts = {
+        .endsWith('\n')
+    );
+  });
 
-test.serial('check before fix', async t => {
-  const { exitCode, stdout } = await execp('npm run check', execOpts);
-  t.deepEqual(exitCode, 1);
-  t.notDeepEqual(stdout.indexOf('prettier reported errors'), -1);
-  t.pass();
-});
+  it('should check before fix', async () => {
+    const { exitCode, stdout } = await execp('npm run check', execOpts);
+    assert.strictEqual(exitCode, 1);
+    assert.notStrictEqual(stdout.indexOf('prettier reported errors'), -1);
+  });
 
-test.serial('fix', async t => {
-  const preFix = fs
-    .readFileSync(`${stagingPath}/kitchen/src/server.ts`, 'utf8')
-    .split('\n');
-  await simpleExecp('npm run fix', execOpts);
-  const postFix = fs
-    .readFileSync(`${stagingPath}/kitchen/src/server.ts`, 'utf8')
-    .split('\n');
-  t.deepEqual(preFix[0].trim() + ';', postFix[0]); // fix should have added a semi-colon
-  t.pass();
-});
+  it('should fix', async () => {
+    const preFix = fs
+      .readFileSync(`${stagingPath}/kitchen/src/server.ts`, 'utf8')
+      .split('\n');
+    await simpleExecp('npm run fix', execOpts);
+    const postFix = fs
+      .readFileSync(`${stagingPath}/kitchen/src/server.ts`, 'utf8')
+      .split('\n');
+    assert.strictEqual(preFix[0].trim() + ';', postFix[0]); // fix should have added a semi-colon
+  });
 
-test.serial('check after fix', async t => {
-  await simpleExecp('npm run check', execOpts);
-  t.pass();
-});
+  it('should check after fix', async () => {
+    await simpleExecp('npm run check', execOpts);
+  });
 
-test.serial('build', async t => {
-  await simpleExecp('npm run compile', execOpts);
-  fs.accessSync(`${stagingPath}/kitchen/build/src/server.js`);
-  fs.accessSync(`${stagingPath}/kitchen/build/src/server.js.map`);
-  fs.accessSync(`${stagingPath}/kitchen/build/src/server.d.ts`);
-  t.pass();
-});
+  it('should build', async () => {
+    await simpleExecp('npm run compile', execOpts);
+    fs.accessSync(`${stagingPath}/kitchen/build/src/server.js`);
+    fs.accessSync(`${stagingPath}/kitchen/build/src/server.js.map`);
+    fs.accessSync(`${stagingPath}/kitchen/build/src/server.d.ts`);
+  });
 
-/**
- * Verify the `gts clean` command actually removes the
- * output dir
- */
-test.serial('clean', async t => {
-  await simpleExecp('npm run clean', execOpts);
-  t.throws(() => {
-    fs.accessSync(`${stagingPath}/kitchen/build`);
+  /**
+   * Verify the `gts clean` command actually removes the
+   * output dir
+   */
+  it('should clean', async () => {
+    await simpleExecp('npm run clean', execOpts);
+    assert.throws(() => fs.accessSync(`${stagingPath}/kitchen/build`));
   });
-});
 
-/**
- * CLEAN UP - remove the staging directory when done.
- */
-test.after.always('cleanup staging', async () => {
-  if (!keep) {
-    stagingDir.removeCallback();
-  }
+  /**
+   * CLEAN UP - remove the staging directory when done.
+   */
+  after('cleanup staging', () => {
```

Comment:
Nit: Should `after` be right after `before`? I think that's our "standard" in other tests. Could go either way though